examples/app/README.md

```diff
@@ -71,7 +71,7 @@ The demos container here are:
   <tr><td colspan="2">selection_histogram</td></tr>
   <tr>
     <td><a href="https://github.com/bokeh/bokeh/tree/master/examples/app/selection_histogram.py"><img src="https://docs.bokeh.org/static/selection_histogram_t.png" width=400></img></a></td>
-    <td>Shows axis histograms for selected <em>and</em> nonselected points in a scatter plot</td>
+    <td>Shows axis histograms for selected <em>and</em> unselected points in a scatter plot</td>
```

Comment:
Just FYI the API has long used "nonselection" to specifically mean "those points which are not selected ***when there is a non-empy selection active***".  This in contrast to when *no* points are selected. It's an important point to distinguish in the API because it's almost always the case that the "default" visual appearance (when there is no selection at all) is different from the appearance of "left over" points that are not selected alongside an active selection, and we used the somewhat specialized term "nonselected" to emphasize this distinction. For purposes of this copy text in a demo, it's probably ok to change to "unselected"